docs/README.md

```diff
@@ -148,4 +148,62 @@ And the following message in Celeborn Worker's log:
 INFO [dispatcher-event-loop-4] Controller: Reserved 1 primary location and 0 replica location for local-1690000152711-0
 INFO [dispatcher-event-loop-3] Controller: Start commitFiles for local-1690000152711-0
 INFO [async-reply] Controller: CommitFiles for local-1690000152711-0 success with 1 committed primary partitions, 0 empty primary partitions, 0 failed primary partitions, 0 committed replica partitions, 0 empty replica partitions, 0 failed replica partitions.
+```
+
+## Start MapReduce With Celeborn
+### Add Celeborn client jar to MapReduce's classpath
+1.Add $CELEBORN_HOME/mr/*.jar to `mapreduce.application.classpath` and `yarn.application.classpath`.
+2.Restart your yarn cluster.
+### Add Celeborn configurations to MapReduce's conf
+Modify `${HADOOP_CONF_DIR}/yarn-site.xml`
+```xml
+<configuration>
+    <property>
+        <name>yarn.app.mapreduce.am.job.recovery.enable</name>
+        <value>false</value>
+    </property>
+
+    <property>
+        <name>yarn.app.mapreduce.am.command-opts</name>
+        <!-- Append 'org.apache.celeborn.mapreduce.v2.app.MRAppMasterWithCeleborn' to this setting  -->
+        <value>org.apache.celeborn.mapreduce.v2.app.MRAppMasterWithCeleborn</value>
+    </property>
+</configuration>
+```
+Modify `${HADOOP_CONF_DIR}/mapred-site.xml`
+```xml
+<configuration>
+    <property>
+        <name>mapreduce.job.reduce.slowstart.completedmaps</name>
+        <value>1</value>
+    </property>
+    <property>
+        <name>mapreduce.celeborn.master.endpoints</name>
+        <!-- Replace placeholder to the real master address       -->
+        <value>placeholder:9097</value>
```

Comment:
If master.port is not the default 9097, the above comment is not accurate enough.

Reply:
Updated.